### PR TITLE
Add 1.7.7.2 release note

### PR DESCRIPTION
**Pull request by MatShir**
Release article to be published when the build is valideted by the QA

Still to-do:
- Clean the issue name
- Add the images

### CODE REVIEW

File: news/_posts/releases/2021-02-22-prestashop-1-7-7-2-maintenance-release.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+After a few months dedicated to fixing upgrades problems, regressions and security issues found on the previous 1.7.7 versions, PrestaShop 1.7.7.2 is finally available! 
+
+![1.7.7.2 is available!](/assets/images/2021/02/1.7.7.2_banner.png)
+
+This maintenance release fixes 21 bugs reported on version 1.7.7.0 and 1.7.7.1 and 2 security issues.
+
+We suggest upgrading your shop quickly in order to benefit from these changes. Of course, don’t forget to make a backup before.
+
+Reminder:  the 1-Click Upgrade module’s latest version is v4.10.0, don’t forget to upgrade it before using it.
```
Comment (matks) [inline news/_posts/releases/2021-02-22-prestashop-1-7-7-2-maintenance-release.md]:
```suggestion
Reminder:  the 1-Click Upgrade module’s latest version is v4.12.0, don’t forget to upgrade it before using it.
```

File: news/_posts/releases/2021-02-22-prestashop-1-7-7-2-maintenance-release.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+*   [Searching and viewing shopping carts in the back office produces an "unexpected error" screen](https://github.com/PrestaShop/PrestaShop/issues/22812)
+
+**Others**
+
+
+
+*   [Installation Error in Persian (Farsi) Language](https://github.com/PrestaShop/PrestaShop/issues/22657)
+*   [Customer Group Discounts are not showned on Your voucher page in the front-office](https://github.com/PrestaShop/PrestaShop/issues/22397)
+
+Read the [full changelog here](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.7.2)]
```
Comment (matks) [inline news/_posts/releases/2021-02-22-prestashop-1-7-7-2-maintenance-release.md]:
```suggestion
Read the [full changelog here](https://github.com/PrestaShop/PrestaShop/releases/tag/1.7.7.2)
```

File: news/_posts/releases/2021-02-22-prestashop-1-7-7-2-maintenance-release.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+*   [The display of the cart rule on FO with free gift + percentage cart rule is incorrect](https://github.com/PrestaShop/PrestaShop/issues/22615)
+*   [The Shipping price not instantly updated on the Order page](https://github.com/PrestaShop/PrestaShop/issues/21267)
+*   [The VAT number is missing in order page](https://github.com/PrestaShop/PrestaShop/issues/22830)
+*   [I cannot search a product when a shop from different Group is selected on create a the Add new order page](https://github.com/PrestaShop/PrestaShop/issues/22460)
+*   [Symfony ContextErrorException Error 500 when debug enabled](https://github.com/PrestaShop/PrestaShop/issues/22231)
+*   [The refresh button is not working on details modal of carts on the customer page](https://github.com/PrestaShop/PrestaShop/issues/22105)
+*   [The gift block is not updated when changing the Address for a discount on the countrie on the Add new Order page](https://github.com/PrestaShop/PrestaShop/issues/22243)
+*   [BO - Orders page - an error is displayed in the console when adding a voucher · Issue #22590 · PrestaShop/PrestaShop](https://github.com/PrestaShop/PrestaShop/issues/22590)
```
Comment (matks) [inline news/_posts/releases/2021-02-22-prestashop-1-7-7-2-maintenance-release.md]:
```suggestion
*   An error is displayed in the console when adding a voucher(https://github.com/PrestaShop/PrestaShop/issues/22590)
```

File: news/_posts/releases/2021-02-22-prestashop-1-7-7-2-maintenance-release.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+tags:
+ - version
+ - maintenance
+ - patch
+ - releases
+ - 1.7.7.x
+ - 1.7
+---
+
+After a few months dedicated to fixing upgrades problems, regressions and security issues found on the previous 1.7.7 versions, PrestaShop 1.7.7.2 is finally available!
```
Comment (marionf) [inline news/_posts/releases/2021-02-22-prestashop-1-7-7-2-maintenance-release.md]:
```suggestion
After a few weeks dedicated to fixing upgrades problems, regressions and security issues found on the previous 1.7.7 versions, PrestaShop 1.7.7.2 is finally available! 
```

File: news/_posts/releases/2021-02-22-prestashop-1-7-7-2-maintenance-release.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+ - releases
+ - 1.7.7.x
+ - 1.7
+---
+
+After a few months dedicated to fixing upgrades problems, regressions and security issues found on the previous 1.7.7 versions, PrestaShop 1.7.7.2 is finally available! 
+
+![1.7.7.2 is available!](/assets/images/2021/02/1.7.7.2_banner.png)
+
+This maintenance release fixes 21 bugs reported on version 1.7.7.0 and 1.7.7.1 and 2 security issues.
```
Comment (marionf) [inline news/_posts/releases/2021-02-22-prestashop-1-7-7-2-maintenance-release.md]:
```suggestion
This patch release fixes 21 regressions reported on version 1.7.7.0 and 1.7.7.1 and 2 security issues.
```

File: news/_posts/releases/2021-02-22-prestashop-1-7-7-2-maintenance-release.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+## Acknowledgments
+
+Contributors to this patch version, from both the Core team and the community at large: 
+
+Ali Shareei, Boubker Bribri, Franck Lefèvre, Ibrahima Sow, Krystian Podemski, Mathieu Ferment, Nesrine Abdmouleh, Pierre Rambaud, Simon Garny, Thomas Baccelli and Valentin Szczupak
+
+Thank you!
+
+A special version co-released by our maintainer Pululu Kinanga (@Pululuk) from [Agence123](https://www.lagence123.com/) and the PrestaShop team. They helped us to deliver this patch version. This is a new step towards the independence of the open-source project.
```
Comment (Progi1984) [inline news/_posts/releases/2021-02-22-prestashop-1-7-7-2-maintenance-release.md]:
```suggestion
A special version co-released by our maintainer Pululu Kinanga ([@Pululuk](https://github.com/Pululuk)) from [Agence123](https://www.lagence123.com/) and the PrestaShop team. They helped us to deliver this patch version. This is a new step towards the independence of the open-source project.
```

**Review by Progi1984 — COMMENTED** [review]
A small comment

**Review by Progi1984 — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Review by ttoine — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @MatShir

**Pull request merged**